feat: add update 134 (#236)
tari-project/tari-dot-com: _updates/2024-09-27-update-134.md

<div align="center">
  <iframe width="560" height="315"  src="https://www.youtube.com/embed/bPG0MdaXwlA" 
    title="Tari Cape Town Episode 3" 
    allow="autoplay; encrypted-media; gyroscope; picture-in-picture; web-share" 
  allowfullscreen>
  </iframe>
</div>

We’d also like to extend our thanks to the broader Tari community - veterans and newcomers alike - who are currently exploring the Universe, putting it through its paces and providing valuable feedback. Since being made available on the 12th September, Tari Universe has seen major improvements and tweaks:

* Multiple improvements to the user interface, whether that’s giving users the option to control their rate of mining, change their localization (currently English, Afrikaans, Turkish or Polish; we’ll be covering how you can add your own localizations in the next update) or provide stats on other related Tari services and applications (such as referrals and gems from [the Tari Airdrop Game](https://airdrop.tari.com).

> * Incorporating community favorite features, such as integrated, simple P2Pool functionality (which we discussed [in our previous update](https://tari.com/updates/2024-07-22-update-133), which delivers smaller but more consistent mining rewards) and merge mining while making it incredibly simple to run for the average user.

Reviewer: ```suggestion
* Incorporating community favorite features, such as integrated P2Pool functionality, which we discussed [in our previous update](https://tari.com/updates/2024-07-22-update-133), and delivers smaller but more consistent mining rewards. 
```

## The Ootle - Tari’s Second Layer

> Ootle has seen a number of significant fixes, optimizations and new features over the past two months. But, in particular, there’s been a lot of focus on handling cross-shard exchanges and improving Ootle's consensus model.

Reviewer: ```suggestion
The Ootle has seen a number of significant fixes, optimizations and new features over the past two months. But, in particular, there’s been a lot of focus on handling cross-shard exchanges and improving the Ootle's consensus model.
```

### Better Consensus

> Fixes and finessing have been the real name of the game over the past two months for Ootle’s consensus model. Whether improving the security of transactions or tackling the incentive structure for securing good behavior, consensus is that consensus is a better place than ever. Some notable fixes includes (but are certainly not limited to):

Reviewer: ```suggestion
Fixes and finessing have been the real name of the game over the past two months for the Ootle’s consensus model. Whether improving the security of transactions or tackling the incentive structure for ensuring good behavior, consensus is that consensus is a better place than ever. Some notable fixes include (but are certainly not limited to):
```

> * Many improvements the transaction creation process, the inclusion of evidence data, reduced validator node setup time and more ([PR 1141](https://github.com/tari-project/tari-dan/pull/1141))

Reviewer: ```suggestion
* Many improvements to the transaction creation process, the inclusion of evidence data, reduced validator node setup time and more ([PR 1141](https://github.com/tari-project/tari-dan/pull/1141))
```

* Deference of proposed transactions with conflicting input versions for multi-shard environments ([PR 1125](https://github.com/tari-project/tari-dan/pull/1125))
* Improvements around evidence data, including optimization and security improvements such as the inclusion of a unique QC ID for auditing purposes. ([PR 1141](https://github.com/tari-project/tari-dan/pull/1141), [PR 1130](https://github.com/tari-project/tari-dan/pull/1130))

## The Tari Protocol

The Tari protocol continues to see improvements and additional features. One that we’d like to highlight in this update is the  work done on the hot/cold wallet functionality.

### Whither Which Wallet?
Building on the new View Key feature [mentioned in our previous update](https://tari.com/updates/2024-07-22-update-133), we’ve added the ability to create read-only wallets with access to the view key of an existing wallet. As a result, it’s now possible to have a cold wallet with the capability to spend AND receive funds as well as act on any transactions that come in: a significant development that removes the need for a hot spending wallet for online transactions and exchanges.

> Previously, Tari required a traditional hot wallet, which had several concerning security implications. Now, with a view-only hot wallet, mining can be conducted while ensuring the seed words - the key to your wallet - are encrypted while mining. We’ve already implemented it in Tari Universe as a standard, ensuring built-in security for the average user.

Reviewer: ```suggestion
Previously, Tari required a traditional hot wallet, which had several concerning security implications. Now, with a view-only hot wallet, mining can be conducted while ensuring the seed words - the key to your wallet - remain encrypted or even offline in a cold wallet. 
```
